src/Tribe/Asset/Admin.php

```diff
@@ -1,21 +1,24 @@
 <?php
 
+
 class Tribe__Events__Asset__Admin extends Tribe__Events__Asset__Abstract_Asset {
+
 	public function handle() {
 		$deps = array_merge(
 			$this->deps,
-			array(
-				'jquery',
-				'jquery-ui-datepicker',
-				'jquery-ui-sortable',
-				'tribe-bumpdown',
-				'underscore',
-				'wp-util',
-			)
+				array(
```

Comment:
Sorry for not noticing this earlier - but the indentation is odd here: the second param should be tabbed back, ie:

```
$array_merge(
    $first_array,
    array (
        ...
    )
);
```